reark/src/test/java/io/reark/reark/utils/RxViewBinderTest.java

```diff
@@ -63,19 +65,18 @@ public void setUp() {
         MockitoAnnotations.initMocks(this);
         // Mocking AndroidSchedulers.mainThread() as loopers are not mocked by android unit tests
         PowerMockito.stub(PowerMockito.method(AndroidSchedulers.class, "mainThread"))
-                    .toReturn(Schedulers.immediate());
+                    .toReturn(Schedulers.single());
 
-        testSubject = PublishSubject.create();
+        testSubject = PublishProcessor.create();
 
         binder = new RxViewBinder() {
             @Override
-            protected void bindInternal(@NonNull final CompositeSubscription compositeSubscription) {
+            protected void bindInternal(@NonNull final CompositeDisposable compositeDisposable) {
                 assertNull(testSubscriber);
 
                 // It is not possible to subscribe multiple with the same subscriber.
                 testSubscriber = new TestSubscriber<>();
-
-                compositeSubscription.add(testSubject.subscribe(testSubscriber));
+                compositeDisposable.add(testSubject.subscribe(testSubscriber::onNext));
```

Comment:
@ulx, this should work if you change to `compositeDisposable.add(testSubject.subscribeWith(testSubscriber));`